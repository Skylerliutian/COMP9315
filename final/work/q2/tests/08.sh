./bnl 100 20 10
