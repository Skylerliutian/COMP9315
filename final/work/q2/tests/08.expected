Running: ./bnl 100 20 10

Frames:     [00] [01] [02] [03] [04] [05] [06] [07] [08] [09]
Contents:    S16  S17  S11  S14  S15  R99  S12  S18  S13  S19
PinCount:      0    0    0    0    0    0    0    0    0    0
Popularity:    2    2    0    1    1    1    0    2    0    2
Clock: 0

#requests: 2100
#releases: 2100
#hits    : 0
#reads   : 2100
